### PR TITLE
Bundles: limit release calendar pages to provisional/confirmed

### PR DESCRIPTION
**Pull request by zerolab**
This builds on #33 and adds a separate chooser for the Bundle's release calendar page chooser that limits the options to only RC pages that are not cancelled or already published.

https://www.loom.com/share/2f26efc2772142af83b3f3beff7b44e6

also ensures chosen pages are deduplicated

### CODE REVIEW

File: cms/release_calendar/viewsets.py [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+            self.title_column,  # type: ignore[attr-defined]
+            Column("release_date"),
+            Column("release_status", label=_("Status"), accessor="get_status_display"),
+            DateColumn(
+                "updated",
+                label=_("Last Updated"),
+                width="12%",
+                accessor="latest_revision_created_at",
+            ),
+            PageStatusColumn("status", label=_("Page status"), width="12%"),
```
Comment (MebinAbraham) [inline cms/release_calendar/viewsets.py:28]: Non-blocker: I found seeing both Status and Page status in the chooser confusing. Is page status important enough to show, it doesn't impact the selection?. Though, that might just be me 🤷

**Review by MebinAbraham — COMMENTED** [review]
Food for thought: This prevents the chooser from showing Cancelled or Published but doesn't restrict those server-side. You could also have selected one that was Confirmed and subsequently cancelled, but no errors would occur. I'm happy for the resolution/consideration of that to be separate.

File: cms/release_calendar/viewsets.py (hunk 1 above)
Comment (zerolab) [reply cms/release_calendar/viewsets.py:28]: I didn't have it originally, but then I though it may be useful, added it. Will drop

Comment (zerolab) [reply cms/release_calendar/viewsets.py]: cc4da27

**Review by MebinAbraham — APPROVED** [review]

**Review by sanjeevz3009 — APPROVED** [review]
Reviewed and locally everything is as expected! LGTM =)

**Pull request merged**